utilities/infra.py

```
@@ -194,8 +196,11 @@ def get_pod_container_error_status(pod: Pod) -> str | None:
 
 def get_not_running_pods(pods: list[Pod], filter_pods_by_name: str = "") -> list[dict[str | None, str]]:
     pods_not_running = []
+    if is_jira_open(jira_id="CNV-82451"):
+        LOGGER.warning(f"Ignoring {KUBEVIRT_CONSOLE_PLUGIN} pods for pod state validations due to CNV-82451")
```

Ruff (0.15.9)

[warning] Logging statement uses f-string

(G004)